App/screens/CredentialOffer.tsx

```diff
@@ -31,123 +34,187 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
     flexDirection: 'column',
-    backgroundColor: CredentialOfferTheme.background,
+    backgroundColor: ColorPallet.brand.primaryBackground,
+  },
+  listItem: {
+    paddingHorizontal: 25,
+    paddingTop: 16,
+    backgroundColor: ColorPallet.brand.secondaryBackground,
+  },
+  label: {
+    ...TextTheme.label,
+  },
+  textContainer: {
+    justifyContent: 'space-between',
+    minHeight: TextTheme.normal.fontSize,
+    paddingVertical: 15,
+    height: 90,
+  },
+  text: {
+    ...TextTheme.normal,
   },
 })
 
 const CredentialOffer: React.FC<CredentialOfferProps> = ({ navigation, route }) => {
+  if (!route || !route.params) {
```

Comment:
Can usually just shorten to: `!route?.params`

Reply:
Fixed